src/main/java/ExitCommand.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * A Command to exit the Duke program.
+ */
+public class ExitCommand extends Command {
+
+    /**
+     * Executes the command to exit the Duke program.
+     *
+     * @param tasks The task list.
+     * @param ui The ui that handles user output.
```

Comment:
This param does not exist anymore and needs to be removed from the Javadoc comment.